clients/unshaded/src/main/java/tachyon/client/LocalBlockOutStream.java

```diff
@@ -160,6 +161,7 @@ public void close() throws IOException {
       mCloser.close();
       mTachyonFS.cacheBlock(mBlockId);
       mClosed = true;
+      mTachyonFS.getClientMetrics().incBlocksWrittenLocal(1);
```

Comment:
Do you need check if mWrittenBytes > 0 here? Only caches the block and increments the metrics if written bytes > 0 ?